packages/codegen-ui/lib/types/view/view.ts

```diff
@@ -29,6 +30,8 @@ export interface StudioView {
   viewConfiguration: ViewConfiguration;
 }
 
+export declare type ViewType = 'Table' | 'collection';
```

Comment:
capital case Collection?

Reply:
see above replies. How do we handle it? We'll have to change 'Table' to 'table' eventually.